pkg/services/ngalert/schedule/registry.go

```diff
@@ -217,6 +217,7 @@ func (r ruleWithFolder) Fingerprint() fingerprint {
 			writeBytes(nil)
 			return
 		}
+		// #nosec G103
```

Comment:
G103: https://securego.io/docs/rules/g103.html
G103: Use of unsafe block
I think this is audited so I am excluding this too